silverstrike/templates/silverstrike/base.html

```diff
@@ -20,162 +20,171 @@
     <link rel="stylesheet" href="{% static 'vendor/css/skin-blue-light.min.css' %}">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,300italic,400italic,600italic">
 
+    {#    For Dates#}
+    <link rel="stylesheet" href="{% static 'silverstrike/css/daterangepicker.css' %}">
```

Comment:
We only need the stylesheet for one back. Use the stylesheets block in the respective template instead